kge/config-default.yaml

```diff
@@ -492,6 +492,10 @@ entity_ranking:
   # mean_reciprocal_rank_filtered_with_test.
   filter_with_test: True
 
+  # Tie detection tolerances i.e. how close should scores be to count them as ties, see torch.isclose for more details.
+  tie_detection_atol: 1e-08
+  tie_detection_rtol: 1e-05
+
```

Comment:
I suggest to group as `tie_handling.type`, `tie_handling.atol` and `tie_handling.rtol` + for backwards compatiblity a rewrite of `tie_handling` (when str) to `tie_handling.type` (see bottom of `config.py`)